recipes/openmpi/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from os import path
+
+
+class OpenmpiConan(ConanFile):
+    name = "openmpi"
+    license = "3-clause BSD license"
+    author = "Conan Community"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "A High Performance Message Passing Library"
+    topics = ("conan", "openmpi", "mpi")
+    homepage = "https://www.open-mpi.org/"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
```

Comment:
```suggestion
    options = {"shared": [True, False], "fPIC": [True, False]}
    default_options = {"shared": False, "fPIC": True}
```

This recipe should have an fPIC option. See https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#kb-h006-fpic-option